app/views/chargeback_rate/show_list.html.haml

```diff
@@ -0,0 +1,3 @@
+#main_div
+  = render :partial => 'layouts/gtl'
+  = render :partial => 'tree' if @tree.present?
```

Comment:
I don't really like this, but as long as we're not using the react component, why not. Maybe you could move the one-liner from the `tree.html.haml` directly here...